File: lang/clojure/skeleton/test/day_21_test.clj
(ns day-21-test
  (:require
   [clojure.test :refer [deftest is testing]]
   [day-21 :refer [part1 part2]]))

(deftest day-21-test
  (testing "sample data"
    (is (= 0 (part1 "../data/21-sample.txt")))
    (is (= 0 (part2 "../data/21-sample.txt")))))

File: years/2015/elixir/test/aoc/day_21_test.exs
defmodule AoC.Day21.Test do
  @moduledoc false

  use ExUnit.Case, async: false

  alias AoC.Day21

  # comment this out to always log to the console
  @moduletag :capture_log

  describe "part 1" do
    test "with sample data" do
      assert Day21.part_1(["../data/21.txt"]) == 1
    end
  end

  describe "part 2" do
    test "with sample data" do
      assert Day21.part_2(["../data/21.txt"]) == 1
    end
  end
end